app/src/main/java/com/hover/stax/transactions/TransactionDetailsFragment.kt

```diff
@@ -49,16 +50,13 @@ class TransactionDetailsFragment : DialogFragment(), Target{
     private var uuid: String? = null
     private var isFullScreen = false
 
+    private lateinit var bottomSheetBehavior: BottomSheetBehavior<RelativeLayout>
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        isFullScreen = requireArguments().getBoolean(IS_FULLSCREEN)
-        if (isFullScreen) setFullScreen()
+        setFullScreen()
```

Comment:
If transaction details will always be shown as full screen, let's convert this to a proper fragment. It can be simplified to remove unnecessary code.